### ISSUE TITLE
RuneLite crashes whenever I plug in my amp

### ISSUE THREAD

**Issue by Ba3henov** [runelite/runelite#14075]
Whenever Runelite is running, plugging in my amp via usb forces RuneLite to close no matter which workaround I try. This doesn't happen with the vanilla client. This is a problem because I want to practice finding the bassline to various in-game soundtracks.

PC specs: Windows 10, Ryzen 5 2600, GeForce GTX 1650
Amp model: Blackstar ID:Core 10 v3

**Comment by Nightfirecat** [comment]
Do you have any `jvm_crash_pid` files in your logs folder?

**Comment by Ba3henov** [comment]
I looked and was unable to find anything. I opened the client.log and launcher.log files and tracked them as I crashed the game; no new lines were added to either file when I plugged in the amp and the game closed.

**Comment by ThePharros** [comment]
> This is a problem because I want to practice finding the bassline to various in-game soundtracks.

This isn't of any help to the RL client issue itself, but in the meantime could you practice using soundtracks off the OSRS Wiki or YouTube?

**Comment by Ba3henov** [comment]
Yeah that's what I've been doing, but I can't do that at the same time as I use RuneLite.
